ros/content.md

```diff
@@ -1,28 +1,99 @@
-# What is [ROS](http://www.ros.org/)?
+# What is [ROS](https://index.ros.org/doc/ros2)?
```

Comment:
I understand the conclusion of merging repos and documentation, but is there a way to specify in some relevant places when the 2 versions vary. I'm thinking expecially here for documentation entry point and core licensing.

Reply:
How about splitting the the to into separate bullet points: https://github.com/docker-library/docs/pull/1381/commits/6d09179854033de0cc43bbf426832b327c6575f5

Reply:
Keeping ros org landing page as main link 966f6ae